src/models/game.ts

```diff
@@ -11,15 +11,15 @@ export const MAX_POINTS = 25;
 export const MAX_PIECES = 6;
 
 class Game extends Model<InferAttributes<Game>, InferCreationAttributes<Game>> {
-  declare id: number;
+  declare id: CreationOptional<number>;
```

Comment:
`id` will be set automatically when doing `MyModel.create({ myArgs });`, but this `CreationOptional<TheType>` is necessary to tell typescript that `id` is not actually needed when creating the model. Same for created/updated timestamps.